UOP1_Project/Assets/Scripts/Inventory/CharacterCustomization.cs

```diff
@@ -0,0 +1,7 @@
+using UnityEngine;
+
+[CreateAssetMenu(fileName = "CharacterCustomization", menuName = "Inventory/Character Customization", order = 51)]
+public class CharacterCustomization : Item
+{
+    // TODO: Add ingredient-specific fields and properties
+}
```

Comment:
Would be nice if these scripts for SOs could be grouped in one folder under Inventory, to separate them from the MonoBehaviours.

Reply:
Same goes for the other SO scripts (Utensils, etc.)

Reply:
Done. Moved Scriptable Objects to its own subfolder . 

@trevorjmcdonald  please Pull https://github.com/trevorjmcdonald/open-project-1/pull/8/commits/abf170dbb9929c939f0c61489275752201fe83f3

pr #8 in your forks.